### PR TITLE
Provide some guidance on the PSLF and other student loan benefit programs

### PR DESCRIPTION
**Pull request by gboone**
Related to #6 cc: @fureigh @mkhandekar

### CODE REVIEW

**Comment by cbrydges** [comment]
@gboone @fureigh @mkhandekar
Can the section I copied below please be reworded or deleted from the write-up?  18F has not decided whether this is something we want to budget for and offer to career hires - so I do not want to give the impression we will or do.    

The federal Student Loan Repayment Program
 +
 +The Student Loan Repayment Program is an option for federal agencies to repay a certain amount of an employee's loans while they are employed. Unfortunately, this requires a 3 year commitment from the employer. Since we are on 2 + 2 year terms, at this time **most 18F employees do not qualify for this program.**

**Comment by gboone** [comment]
Sure, I didn't realize 18F would have to budget for that. I was aiming to clarify that most of us can't qualify even if it were budgeted for but happy to take it out if it's more confusing than helpful.

**Comment by cbrydges** [comment]
Thanks Greg - we just found out about the budgeting part recently.  I appreciate you clarifying it so candidates/staff do not repeatedly ask about it.  Would it be bad comms to say 18F does not currently offer this benefit?  Worded a bit better of course.

**Comment by gboone** [comment]
I'll see what I can do. Thanks for the clarification :)

**Comment by cbrydges** [comment]
Thanks greg - how about the following so we have more of the information.  Also, it has to be done as part of hiring before an employee starts - so wondering if we should use the word candidate instead.  Am I just making things more complicated?  Please tell me if I am :)     

The Student Loan Repayment Program is an option for federal agencies to repay a certain amount of an employee's loans while they are employed. Unfortunately, this requires a 3 year commitment from the employee. Since we are on 2 + 2 year terms, at this time **most 18F employees do not qualify for this program.**  As such 18F is not offering this benefit to any candidates.

**Comment by gboone** [comment]
I think that's great. I'll add a link to the OPM page about the SRLP, too.

**Comment by cbrydges** [comment]
thank you!

**Comment by rdpiazza** [comment]
@cbrydges @gboone is this OK to be merged/published? If so, I'd like to reference the policy.  Thanks!

**Comment by cbrydges** [comment]
I believe it is ready to be published.

**Comment by rdpiazza** [comment]
OK if I merge then?  (Asking because I don't usually merge in this repo)

**Comment by cbrydges** [comment]
Fine with me but copying @fureigh since I don't usually approve merges, etc. either.

Thanks all!

**Comment by rdpiazza** [comment]
See also #324

**Comment by rdpiazza** [comment]
Pinging @fureigh.  Fureigh -- did you want to review and commit, or should we go ahead?  Thanks!

**Comment by ErieMeyer** [comment]
Hi all! 

There's a miscommunication here. The Student Loan Repayment Programs that are paid for by the agencies are _separate_ from Public Service Loan Forgiveness. Here's more about how PSLF works: http://www.consumerfinance.gov/askcfpb/641/what-public-service-loan-forgiveness.html

ANY public service employee (any level of gov, any nonprofit, etc.) is eligible (depending on their student loan burden) for the PSLF program.

The TLDR is that the only thing that an employer needs to do for an employee to receive PSLF benefits is to _confirm that the employee works for them._ There's no cost for the employer.

The _mega cool_ thing for 18F (and really all of GSA) to do would be to take this simple pledge: http://www.consumerfinance.gov/pledge/

And pre-fill the employer form for all of your employees!

**Comment by gboone** [comment]
I think that's what we have isn't it? I do notice a typo but here's the language we have [in the pre- and post-18F section](https://github.com/18F/joining-18f/blob/98b5b4f446ffb0b5130b20c7a6ebddcc317c7089/pages/student-loan-benefits.md#pre--and-post-18f-work):

> You don't need to work for the federal government to qualify for the PSLF. If you came here from a non-profit, school, or [any other qualifying employer](https://studentaid.ed.gov/sa/repay-loans/forgiveness-cancellation/public-service#qualifying-employment) make sure you get your tenure there to count as well. Likewise, if you go to work for one of those places after your term ends, make sure to re-apply through your new employer.

Can you let us know what language needs to change? Also, thanks for the head's up on the pledge. We should _definitely_ see if GSA will do that for us.

We had to clarify in the Student Loan Repayment Program section that the SRLP is likely not something 18F employees will likely be able to qualify for since we cannot make a 3 year commitment and 18F does not, right now have the budget for it anyway.

**Comment by fureigh** [comment]
Sorry for the delay, @cbrydges and @rdpiazza; I think I missed those notifications when GitHub-to-Slack notifications were down.

Looks like @gboone merged the previous PR but I'll keep an eye out for the next one.

**Comment by fureigh** [comment]
By which I mean — looks like @gboone merged #324, but lmk when this one's ready for review. Thanks!

**Comment by gboone** [comment]
I think this one's ready to go? I don't know.

**Comment by fureigh** [comment]
Thanks, y'all! Pardon the delayed reply; the removal of the Github-to-Slack notification bot meant I didn't see some of these.

I made some copy edits and added the necessary remaining metadata so the page will actually show up. I also added a section and linked to it from the [Benefits page](https://pages.18f.gov/joining-18f/benefits/).

If anyone has additional language changes, please open a new pull request and mention this one in the description. Thanks for all of your work!
